### DISCUSSION TITLE
Does networkx 2.5.1 officially support Python 3.9?

### DISCUSSION THREAD

**Discussion by justinjdickow** [networkx/networkx#4782] [Q&A] [answered]
I see it in the readme as 3.6|3.7|3.8 but in travis as being built against 3.9 as well.

**Comment by MridulS** [comment, marked as answer]
"official" support for 3.9 was added in https://github.com/networkx/networkx/pull/4303 and networkx 2.6 will officially support python3.9

But 2.5.1 works perfectly fine with py3.9, we didn't have to make any changes to make it work with py3.9 (as far as I remember).
